sdk/resourcemanager/Azure.ResourceManager.Core/src/ArmClient.cs

```diff
@@ -145,6 +145,15 @@ public ResourceGroupOperations GetResourceGroupOperations(ResourceGroupResourceI
             return new ResourceGroupOperations(new SubscriptionOperations(new ClientContext(ClientOptions, Credential, BaseUri, Pipeline), id.SubscriptionId), id.ResourceGroupName);
         }
 
+        /// <summary>
+        /// Gets the tags operations under this subscription.
+        /// </summary>
+        /// <returns> The tags operations. </returns>
+        public virtual TagsOperations GetTagOperations()
+        {
+            return new TagsOperations(new ClientContext(ClientOptions, Credential, BaseUri, Pipeline), DefaultSubscription.Id.SubscriptionId);
```

Comment:
A tag operation would need to take in a tag id not the parent Id

Reply:
Done